src/main/js/field/input/GeneralButton.js

```diff
@@ -1,8 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const GeneralButton = ({ onClick, children, className, id }) => (
-    <button id={id} className={`btn btn-primary ${className}`} type="button" onClick={onClick}>{children}</button>
+const GeneralButton = ({
```

Comment:
Was this done on purpose for readability?

Reply:
This change was actually made after some updates to our JS sources and NPM. Not sure exactly what introduced this, but it seems to be the recommended way from ESLint